.sample_configs/param_handlers/create_training_pipeline_tabular_forecasting_sample.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+def make_parent(parent: str) -> str:
+    parent = parent
+
+    return parent
+
+def make_training_pipeline(
+            display_name: str, 
+            dataset_id: str, 
+            model_display_name: str, 
+            target_column: str, 
+            time_series_identifier_column: str,
+            time_column: str,
+            static_columns: str,
+            time_variant_past_only_columns: str,
+            time_variant_past_and_future_columns: str,
+            forecast_window_end: int,
+            ) -> google.cloud.aiplatform_v1alpha1.types.training_pipeline.TrainingPipeline:
+    # set the columns used for training and their data types
+    transformations = [
+        {"auto": {"column_name": "date"}},
+        {"auto": {"column_name": "state_name"}},
+        {"auto": {"column_name": "county_fips_code"}},
+        {"auto": {"column_name": "confirmed_cases"}},
+        {"auto": {"column_name": "deaths"}}
+    ]
+
+    period = {"unit": "day", "quantity": 1}
+
+    training_task_inputs_dict = {
+        # required inputs
+        "targetColumn": target_column,
```

Comment:
Curious as to if we should show comments for each of these values in the sample.

Reply:
@ivanmkc do you mean comments like # display_name: YOUR_DISPLAY_NAME?

Reply:
Thanks for reviewing, I was wondering if we should add comments for each param in the samples.

Seems like Yuhan is suggesting to just tell them to read the docstrings.